pleskdistup/common/src/util.py

```diff
@@ -9,31 +9,59 @@
 # Returns standard output
 def logged_check_call(cmd: typing.Union[typing.Sequence[str], str], **kwargs) -> str:
     log.info(f"Running: {cmd!r}. Output:")
+    stdout = []
+
+    def proc_stdout(line: str) -> None:
+        log.info(f"stdout: {line}", to_stream=False)
+        stdout.append(line)
+
+    def proc_stderr(line: str) -> None:
+        log.info(f"stderr: {line}")
 
-    # I beleive we should be able pass argument to the subprocess function
-    # from the caller. So we have to inject stdout/stderr/universal_newlines
-    kwargs["stdout"] = subprocess.PIPE
-    kwargs["stderr"] = subprocess.STDOUT
+    exit_code = exec_get_output_streamed(cmd, proc_stdout, proc_stderr, **kwargs)
+    if exit_code != 0:
+        log.err(f"Command {cmd!r} failed with return code {exit_code}")
+        raise subprocess.CalledProcessError(returncode=exit_code, cmd=cmd)
```

Comment:
@vizovitin @SandakovMM I deliberately removed `output = ''.join(stdout)` in `CalledProcessError`. It's really a pain when stdout is very large and program exits with non-zero (i.e. the leapp binary); then we read same output twice. IMO it doesn't make sense (sending all stdout in exception) as well.

Reply:
OK, although I don't really see a real problem here.

Reply:
Main problem for me is reading the logs :)